canonical-web-and-design/maas-ui
# Build set pool form in action menu.

## Done
- Built set pool form in action menu
- Note that the "Create pool" radio button has been disabled until issue #928 has been resolved

## QA
- Select some machines and choose "Set resource pool..." from the action menu
- Check that you can set the machines' pools to it

## Fixes
Fixes canonical-web-and-design/MAAS-squad#1789

## Screenshot
![Screenshot_2020-04-01 Machines karura MAAS](https://user-images.githubusercontent.com/25733845/78108194-3e473280-743a-11ea-90f7-0e33b25e4ff7.png)
